src/pretix/presale/templates/pretixpresale/organizers/customer_memberships.html

```diff
@@ -0,0 +1,110 @@
+{% extends "pretixpresale/organizers/customer_base.html" %}
+{% load i18n %}
+{% load icon %}
+{% load eventurl %}
+{% load textbubble %}
+{% block title %}{% trans "Memberships" %}{% endblock %}
+{% block inner %}
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <h3 class="panel-title">
+                {% icon "id-badge" %}
+                <b>{% trans "Memberships" %}</b> ({{ page_obj.paginator.count }})
+            </h3>
+        </div>
+        <div class="panel-body">
+            {% if memberships %}
+                <ul class="full-width-list alternating-rows">
+                    {% for m in memberships %}
+                    <li class="row">
+                        <dl>
+                            <div class="col-xs-5">
+                                <dt>
+                                    {% if m.canceled %}<del>{% endif %}
+                                    <a href="{% abseventurl request.organizer "presale:organizer.customer.membership" id=m.id %}">
+                                        {{ m.membership_type.name }}
+                                    </a>
+                                    {% if m.canceled %}</del>{% endif %}
+                                    {% if m.membership_type.transferable %}
+                                        <span class="text-muted" data-toggle="tooltip" title="{% trans "Membership is transferable" %}">
+                                            {% icon "users" %}
+                                        </span>
+                                    {% endif %}
+                                </dt>
+                                <dd class="text-muted">
+                                    {% if not m.membership_type.transferable %}
+                                        {% icon "id-badge" %}
+                                    {% endif %}
+                                    {{ m.attendee_name|default_if_none:"" }}
```

Comment:
Either add some text for the default_if_none case, or remove the whole line, otherwise it looks a bit broken /unclear what the icon means without attendee name:
![image](https://github.com/user-attachments/assets/423f1899-0cd8-433a-8cf8-b6213bf5e536)

Reply:
Good catch – in my tests, when it is not transferable, I always had a name when membership was not transferable. Not sure how you managed to get in that state? Ah, maybe changing the membership later! I will fix as you suggest. :)